src/languages/es.ts

```diff
@@ -3784,6 +3784,9 @@ export default {
         keepThisOne: 'Mantener éste',
         hold: 'Bloqueado',
     },
+    reportViolations: {
```

Comment:
Did we get this translation checked? Also this has a period whereas the english version does not

Reply:
There were confirmed [here](https://swmansion.slack.com/archives/C01GTK53T8Q/p1719561220095539)